oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -211,7 +242,12 @@ protected GoogleCredentials() {
    * @param accessToken initial or temporary access token
    */
   public GoogleCredentials(AccessToken accessToken) {
-    super(accessToken);
+    this(accessToken, null);
+  }
+
+  public GoogleCredentials(Builder builder) {
```

Comment:
Does this need to be public?

Reply:
Made it protected

Reply:
This new constructor still looks public

Reply:
nvm, I was looking at an old commit I guess